test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml

```diff
@@ -0,0 +1,25 @@
+test_duration: 70
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=250 -pop seq=1..30000000"
```

Comment:
Need to remove all the 'jmx' options

Reply:
again since it's going to older branch that might not have the change that removed all those, it's might be better to match, to avoid confusion when comparing to the yaml originated this one.